execution-engine/contracts/hdac-system/pop/src/contract_delegations.rs

```diff
@@ -102,6 +103,44 @@ impl ContractDelegations {
             runtime::put_key(&name, Key::Hash([0; 32]));
         }
     }
+
+    pub fn read_user_stat() -> Result<DelegationStat> {
+        let mut delegation_stat = BTreeMap::new();
+        for (name, _) in runtime::list_named_keys() {
+            let mut split_name = name.split('_');
+            if Some("d") != split_name.next() {
+                continue;
+            }
+
+            let to_publickey = |hex_str: &str| -> Result<PublicKey> {
```

Comment:
@jhyeom26 
This lambda function(?) is used frequently.
Should it be replaced as a library function?